package.json

```diff
@@ -16,8 +16,8 @@
   ],
   "license": "AGPL-3.0",
   "engines": {
-    "node": ">=18.12.0",
-    "npm": ">=8.19.2"
+    "node": ">=22.4.0",
```

Comment:
I think you need to update the commit message to say node 22 and also make it a major.
You can probably also bump tsconfig.browser.json's target to es2019 like we have it in the UI.